package/MDAnalysis/transformations/__init__.py

```diff
@@ -26,34 +26,72 @@
 Trajectory transformations --- :mod:`MDAnalysis.transformations`
 ================================================================
 
-The transformations submodule contains a collection of functions to modify the
-trajectory. Coordinate transformations, such as PBC corrections and molecule fitting
-are often required for some analyses and visualization, and the functions in this
-module allow transformations to be applied on-the-fly.
-These transformation functions can be called by the user for any given
-timestep of the trajectory, added as a workflow using :meth:`add_transformations`
-of the :mod:`~MDAnalysis.coordinates.base` module, or upon Universe creation using
+The transformations submodule contains a collection of function-like classes to
+modify the trajectory.
+Coordinate transformations, such as PBC corrections and molecule fitting
+are often required for some analyses and visualization, and the functions in
+this module allow transformations to be applied on-the-fly.
+
+A typical transformation class looks like this:
+
+.. code-blocks:: python
+
+    class transformation(object):
```

Comment:
It's kind of odd to have class names lowercase, normally this is uppercase for python, but I get that it's to preserve backwards compatibility

Reply:
Yeah I was hesitating about that. But I guess it's fine to use lowercase for callable classes according to PEP 8:
>Class Names
Class names should normally use the CapWords convention.
The naming convention for functions may be used instead in cases where the interface is documented and used primarily as a callable.

And for backwards compatibility of course.

Reply:
Especially with the 
> The naming convention for functions may be used instead in cases where the interface is documented and used primarily as a callable.
I am totally ok with lower case.